draft-ietf-quic-bit-grease.md

```diff
@@ -36,17 +36,17 @@ arbitrary value for the second-most significant bit in QUIC packets.
 
 # Introduction
 
-QUIC {{!QUIC=RFC9000}} intentionally describes a very narrow set of fields that
-are visible to entities other than endpoints.  Beyond those characteristics that
-are defined as invariant {{?QUIC-INVARIANTS=RFC8999}}, very little about the
-"wire image" {{?RFC8546}} of QUIC is visible.
+QUIC Invariants {{!QUIC-INV=RFC8999}} intentionally describes a very narrow set
```

Comment:
I know you probably used `QUIC-INV` on purpose but it makes it less consistent with a how we reference it in RFC 9000. It's editorial, you decide, just giving some 2c

Reply:
Yeah, I took another go at it in a later commit after realizing that.

Reply:
Oh deep I looked only at 1 commit